packages/dds/tree/src/core/edit-manager/editManager.ts

```diff
@@ -234,6 +234,13 @@ export class EditManager<
 		return this.trunk;
 	}
 
+	/**
+	 * @returns the head commit of the local branch
+	 */
+	public getLocalBranch(): GraphCommit<TChangeset> {
```

Comment:
Would a name like `getLocalBranchHead` make more sense here?

Reply:
Yea that's probably better.